### PR TITLE
(feat) wrap docker run_command in bash

### PR DESCRIPTION
**Pull request by tphoney**

### CODE REVIEW

**Comment by adreyer** [comment]
The solution for this is going to be to add a new `shell_command` option to the docker transport. When set all commands on the target will be wrapped in the `shell_command`

File: lib/bolt/transport/docker.rb [hunk 1]
```diff
@@ -8,7 +8,7 @@ module Bolt
   module Transport
     class Docker < Base
       def self.options
-        %w[host service-url service-options tmpdir interpreters]
+        %w[host service-url service-options tmpdir interpreters shell_command tty]
```
Comment (adreyer) [inline lib/bolt/transport/docker.rb]: Note that `service-url` has a hyphen, this probably should too for consistency in the config file.

**Comment by adreyer** [comment]
The new config options should be added to https://github.com/puppetlabs/bolt/blob/master/pre-docs/bolt_configuration_options.md#docker-transport-configuration-options

**Comment by florindragos** [comment]
> Can you update the commit message to include the ticket number.

@adreyer Done

**Pull request merged**